core/nodepool/config/templates/stack-template.json

```diff
@@ -361,6 +368,7 @@
                   "Action": [
                     "autoscaling:DescribeAutoScalingGroups",
```

Comment:
For an usual use-case, running CA in a controller node is recommended and that's the default setup.